Source/WebKit/Platform/IPC/Connection.cpp

```diff
@@ -461,7 +461,8 @@ void Connection::removeMessageReceiver(ReceiverName receiverName, uint64_t desti
     removeMessageReceiveQueue(ReceiverMatcher::createWithZeroAsAnyDestination(receiverName, destinationID));
 }
 
-void Connection::dispatchMessageReceiverMessage(MessageReceiver& messageReceiver, UniqueRef<Decoder>&& decoder)
+template<typename MR>
```

Comment:
`MR` is not a good abbreviation. I think we should use `MessageReceiver` or `MessageReceiverType`